met-web/src/components/comments/admin/reviewListing/index.tsx

```diff
@@ -129,9 +129,7 @@ const CommentListing = () => {
             <Grid item xs={0} md={4} lg={4}></Grid>
 
             <Grid item xs={12}>
-                <Typography variant="h4" sx={{ fontWeight: 'bold' }}>
-                    {`${comments[0]?.survey || ''} Comments`}
-                </Typography>
+                <MetHeader1 sx={{ fontWeight: 'bold' }}>{`${comments[0]?.survey || ''} Comments`}</MetHeader1>
```

Comment:
I see a lot of entries where the fontWeight or other styles are getting changed anyway, this goes against the standardization, If the hearders are going to look the same in the whole app then these attributes should be incorporated in the component itself.

Reply:
Good point, but I think the BC standardization isn't setting the bold option here for all h1s. Will MetHeader1 always be bold? Maybe let's confirm with MA @djnunez-aot

Reply:
Font weight 500 is the header standard with Marie Anik's design